DoodleMaze/app/src/main/res/values/strings.xml

```diff
@@ -32,7 +33,17 @@
     <string name="Canada">Canada by Pictures of the Floating World</string>
     <string name="Elevator">Elevator by Pictures of the Floating World</string>
     <string name="Memories">Memories by Pictures of the Floating World</string>
-    <string name="Images">Images/Videos</string>
+    <string name="Images">Images</string>
+    <string name="Video">Video</string>
+    <string name="Audio">Audio</string>
     <string name="Back">Back</string>
+    <string name="img1">https://pixabay.com/photos/labyrinth-freedom-orientation-4300600</string>

```

Comment:
Thanks for adding these.  Would you mind writing these up in a way that gives the creator and title (where possible)?  I need to go back and add the links for the other ones tonight also.  One last thing to do during the final review before turn-in.

For example: 
Labyrinth Image by mwewering
Link: https://pixabay.com/photos/labyrinth-freedom-orientation-4300600

Reply:
Done updating the Credits page